ci-operator/step-registry/openshift/powervc/podman/openshift-powervc-podman-commands.sh

```
@@ -0,0 +1,616 @@
+#   PVCCTL_VERSION     – (in)  tag of the quay.io/powercloud/pvcctl image to run.
+#   HOME               – (in)  private directory set by install_required_tools;
+#                              used for the containers auth.json location.
+#   REGISTRY_AUTH_FILE – (out) set so podman reads the auth.json written above.
+# Returns:
+#   0 on success; exits non-zero if the pvcctl run fails, config.yaml is not
+#   produced, or the PowerVC URL cannot be resolved.
+#######################################
+function setup_config_yaml() {
+	log_info "Setting up config.yaml..."
+
+	cd ${HOME} || {
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)